astroquery/ipac/irsa/core.py

```diff
@@ -122,6 +128,42 @@ def query_sia(self, *, pos=None, band=None, time=None, pol=None,
 
     query_sia.__doc__ = query_sia.__doc__.replace('_SIA2_PARAMETERS', SIA2_PARAMETERS_DESC)
 
+    def query_ssa(self, *, pos=None, diameter=None, band=None, time=None, format=None,
+                  collection=None):
+        """
+        Use standard SSA attributes to query the IRSA SSA service.
+
+        Parameters
+        ----------
+        pos : `~astropy.coordinates.SkyCoord` class or sequence of two floats
+            the position of the center of the circular search region.
+            assuming icrs decimal degrees if unit is not specified.
+        diameter : `~astropy.units.Quantity` class or scalar float
```

Comment:
why diameter instead of radius?  radius is more common across the rest of our API

Reply:
inherited from SSA API, but I agree that I need to change this to be radius instead.